src/components/Transactions/Transfer/TransferAddModal.js

```diff
@@ -0,0 +1,78 @@
+import React, { useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { get, sortBy } from 'lodash';
+
+import { useShowToast } from '@folio/stripes-acq-components';
+import {
+  stripesConnect,
+  stripesShape,
+} from '@folio/stripes/core';
+
+import TransferAddModalForm from './TransferAddForm';
+import {
+  budgetResource,
+  fundsResource,
+  transactionResource,
+} from '../../../common/resources';
+
+const TransferAddModal = ({
+  budgetName,
+  fiscalYearId,
+  mutator,
+  onClose,
+  resources,
+  stripes,
+}) => {
+  const showCallout = useShowToast();
+
+  const saveTransfer = useCallback(
+    async (formValue) => {
+      try {
+        const transfer = await mutator.transaction.POST({
+          ...formValue,
+          fiscalYearId,
+          transactionType: 'Transfer',
+          currency: stripes.currency,
+          source: 'User'
```

Comment:
magic string. Please look if there already a constant for it, or create such

Reply:
fixed